common/utils.robot

```diff
@@ -72,3 +75,26 @@ Create Suite Directory
 Get NOW Time
     ${year}  ${month}  ${day}  ${hour}  ${min}  ${sec}   Get Time   year month day hour min sec
     [Return]   ${year}${month}${day}_${hour}${min}${sec}
+
+Get File Location From Variable
+    ${file.path}   Get From Dictionary   ${file}   path
+    ${file.basename}   Get From Dictionary   ${file}   basename
+    [Return]   ${file.path}/${file.basename}   ${file.path}   ${file.basename}
```

Comment:
I guess so, but probably I should modify some Read File test as

https://github.com/indigo-iam/escape-auth-tests/blob/issue-9/test/datalake/group_based_authz/step_0.robot#L25

because here I check a known file content.
Should I implement usage of temp file anyway?